client/test/e2e/composer/views/model-editor-view.js

```diff
@@ -56,20 +69,71 @@ var ModelEditorView = (function () {
     this.getCurrentModelName = function() {
       return this.modelNameInput.getAttribute('value');
     };
+    this.getCurrentDataSourceIndex = function() {
+      return this.dataSourceSelect.getAttribute('value');
+    };
     this.createNewModel = function(modelName) {
       // TODO: these sleeps shouldn't be necessary..
-      browser.sleep(500);
-      browser.executeScript('window.scroll(0, 0);');
-      browser.sleep(500);
-      browser.wait(EC.visibilityOf(this.modelNameInput), 10000);
-      this.modelNameInput.click();
+      //browser.sleep(500);
```

Comment:
No need to commit commented out code, just delete it if you've solved the problem.